openshift_images/managing_images/using-image-pull-secrets.adoc

```diff
@@ -12,7 +12,7 @@ However, for other scenarios, such as referencing images across {product-title}
 
 You can obtain the image pull secret, `pullSecret`, from the link:https://console.redhat.com/openshift/install/pull-secret[Pull Secret] page on the {console-redhat-com} site.
 
-You use this pull secret to authenticate with the services that are provided by the included authorities, including link:quay.io[Quay.io] and link:registry.redhat.io[registry.redhat.io], which serve the container images for {product-title} components.
+You use this pull secret to authenticate with the services that are provided by the included authorities, from the link:https://quay.io/[Quay.io] and from the link:https://registry.redhat.io[registry.redhat.io], which serve the container images for {product-title} components.
```

Comment:
The links work fine but I'm not crazy about 'from the.' I think it reads better as follows: 

You use this pull secret to authenticate with the services that are provided by the included authorities from link:https://quay.io/[Quay.io] and link:https://registry.redhat.io[registry.redhat.io], which serve the container images for {product-title} components.

Reply:
Removed "from the".